voice/asr-inbound-call.py

```diff
@@ -0,0 +1,43 @@
+#!/usr/bin/env python3
```

Comment:
Can we name the file `handle-user-input-with-asr.py` so that it's easier to match up to the code snippet documentation?